Prometheus/Summary.cs

```diff
@@ -96,30 +92,23 @@ internal Child(Summary parent, LabelSequence instanceLabels, LabelSequence flatt
 
                 _headStream = _streams[0];
 
+                _quantileLabels = new Tuple<byte[], byte[]>[_objectives.Count];
                 for (var i = 0; i < _objectives.Count; i++)
                 {
                     _sortedObjectives[i] = _objectives[i].Quantile;
+                    _quantileLabels[i] = TextSerializer.EncodeSystemLabelValue(_objectives[i].Quantile);
                 }
 
                 Array.Sort(_sortedObjectives);
-
-                _sumIdentifier = CreateIdentifier("sum");
-                _countIdentifier = CreateIdentifier("count");
-
-                _quantileIdentifiers = new byte[_objectives.Count][];
-                for (var i = 0; i < _objectives.Count; i++)
-                {
-                    var value = double.IsPositiveInfinity(_objectives[i].Quantile) ? "+Inf" : _objectives[i].Quantile.ToString(CultureInfo.InvariantCulture);
-
-                    _quantileIdentifiers[i] = CreateIdentifier(null, "quantile", value);
-                }
             }
 
-            private readonly byte[] _sumIdentifier;
-            private readonly byte[] _countIdentifier;
-            private readonly byte[][] _quantileIdentifiers;
+            private readonly Tuple<byte[], byte[]>[] _quantileLabels;
+            private static readonly byte[] SumLabelBytes = PrometheusConstants.ExportEncoding.GetBytes("sum");
```

Comment:
To me a "label" is the name-value pair, so I would expect to see both in a variable named `SumLabelBytes`. Perhaps `SumIdentifierBytes` or `SumNameBytes` is better if this is only the name of the label.

Reply:
I've canocalised the convention `*Suffix` and `*LabelName`.